src/frontend/js/pages/TeacherTrainingDashboardLoader/index.tsx

```diff
@@ -0,0 +1,113 @@
+import CourseRunList from 'pages/TeacherCourseDashboardLoader/CourseRunList';
+import { DashboardLayout } from 'widgets/Dashboard/components/DashboardLayout';
+import { TeacherCourseDashboardSidebar } from 'widgets/Dashboard/components/TeacherCourseDashboardSidebar';
+import { Spinner } from 'components/Spinner';
+import { DashboardCard } from 'widgets/Dashboard/components/DashboardCard';
+import { Icon, IconTypeEnum } from 'components/Icon';
+import Banner, { BannerType } from 'components/Banner';
+import { useCourseProductRelation } from 'hooks/useCourseProductRelation';
+
+const messages = defineMessages({
+  pageTitle: {
+    defaultMessage: 'Training area',
+    description: 'Use for the page title of the training area',
+    id: 'components.TeacherTrainingDashboardLoader.pageTitle',
+  },
+  loading: {
+    defaultMessage: 'Loading training...',
+    description: 'Message displayed while loading a course',
+    id: 'components.TeacherTrainingDashboardLoader.loading',
+  },
+  errorNoProduct: {
+    defaultMessage: "This product doesn't exist",
+    description: 'Message displayed when requested product is not found',
+    id: 'components.TeacherTrainingDashboardLoader.errorNoProduct',
+  },
+});
+
+export const TeacherTrainingDashboardLoader = () => {
+  const intl = useIntl();
+  const { courseProductRelationId } = useParams<{
+    courseProductRelationId: string;
+  }>();
+
+  const {
+    item: courseProductRelation,
+    states: { fetching },
+  } = useCourseProductRelation(courseProductRelationId);
+
+  return (
+    <DashboardLayout sidebar={<TeacherCourseDashboardSidebar />}>
+      <div className="dashboard__page_title_container">
+        <h1 className="dashboard__page_title">
+          <FormattedMessage {...messages.pageTitle} />
+        </h1>
+      </div>
+      {fetching && (
+        <Spinner aria-labelledby="loading-courses-data">
+          <span id="loading-courses-data">
+            <FormattedMessage {...messages.loading} />
+          </span>
+        </Spinner>
+      )}
+
+      {!fetching && !courseProductRelation && (
+        <Banner
+          message={intl.formatMessage(messages.errorNoProduct)}
+          type={BannerType.ERROR}
+          rounded
+        />
+      )}
+
+      {!fetching && courseProductRelation && (
+        <div className="teacher-course-page">
+          <DashboardCard
+            className="icon-arrow-right-rounded"
+            header={
+              <div>
+                <div className="dashboard__title_container_h1">
+                  <h2 className="dashboard__title_h1">
+                    {capitalize(courseProductRelation.product.title)}
+                  </h2>
+                </div>
+                <div className="dashboard__quote">
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                  incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+                </div>
```

Comment:
The product description can be blank, we should manage this case.
```suggestion
{ courseProductRelation.product?.description &&
                <div className="dashboard__description">
                  {courseProductRelation.product?.description}
                </div>
}
```